src/utils/common.js

```diff
@@ -229,6 +222,9 @@ export const formatAmount = (amount: string | number, precision: number = 6): st
   return new BigNumber(roundedNumber).toFixed(); // strip trailing zeros
 };
 
+export const getDisplayedTokenAmount = (amount: string | number, assetSymbol: ?string): string =>
```

Comment:
```suggestion
export const formatTokenAmount = (amount: string | number, assetSymbol: ?string): string =>
```
Consider renaming to improve readability (getXxx may suggest some computation, while formatXxx is very clear)